src/client/dss.py

```diff
@@ -19,6 +21,10 @@
 # Set this value to 1 for debugging
 DSS_DEBUG = 0
 
+# valid list of actions 
+
+dss_valid_actions = ['ls', 'cp', 'rm', 'mb', 'rb', 'hb', 'ho']
```

Comment:
Low pri Suggestion: This should be in dss_info and its name should be dss_valid_action_list to avoid confusion with function valid_dss_action.